### PR TITLE
Micronaut cache extraction

### PR DESCRIPTION
**Pull request by alvarosanchez**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -31,6 +31,7 @@ apply plugin: "io.sdkman.vendors"
 apply plugin: 'org.aim42.htmlSanityCheck'
 
 repositories {
+    mavenLocal()
```
Comment (graemerocher) [inline build.gradle]: Please remove mavenLocal()

File: build.gradle [hunk 2]
```diff
@@ -572,6 +578,7 @@ subprojects { Project subproject ->
 
 
     repositories {
+        mavenLocal()
```
Comment (graemerocher) [inline build.gradle]: Please remove mavenLocal()

File: src/main/docs/guide/aop/caching.adoc [hunk 3]
```diff
@@ -1,61 +1,2 @@
-Similar to Spring and Grails, Micronaut provides a set of caching annotations within the link:{api}/io/micronaut/cache/package-summary.html[io.micronaut.cache] package.
```
Comment (graemerocher) [inline src/main/docs/guide/aop/caching.adoc]: Maybe leave this line at least and the new link to the docs

File: build.gradle (hunk 1 above)
Comment (alvarosanchez) [reply build.gradle]: Yeah, sorry, left it from the local testing I was doing. Fixed now.

**Comment by graemerocher** [comment]
There are conflicts

**Pull request merged**